src/sections/UpcomingEventsSection.js

```diff
@@ -107,56 +176,67 @@ export default function UpcomingEventsSection() {
           </button>
           {showModal && (
             <div className={styles2.container}>
-              <ComputerWindow className={styles2.window} topbarColor="#FB79C3">
-                <div className={styles2.eventInfo}>
-                  <h3 className={styles2.title}>Open Office</h3>
-                  <div className={styles.modalContainer}>
+              <ComputerWindow className={styles2.window} topbarColor="#E2626A">
```

Comment:
is there a way to use the var here instead of specifically setting the hex value